api/routes/events/events.ts

```diff
@@ -41,6 +41,20 @@ router.get(
     res: core.Response<getResponsesBody<GetEvents>, {}, getHTTPCode<GetEvents>>,
   ) => {
     try {
+      // Pagination
+      const start =
```

Comment:
can you name that page ? or skip.

Reply:
As i said, it's not a page but an start ( offset)